exporters/holstebro_helpers.py

```diff
@@ -0,0 +1,811 @@
+        ou_manager = self.mh.read_ou_manager(ou_uuid, False)
+
+        if manager_uuid != None:
+            if ou_manager == {}:  # no manager, create it
+                logger.info("Manager for {} should be {}".format(
+                    ou['name'], manager['Navn']))
+
+                manager_level = self._get_org_level(ou)
+                if has_engagements:
+                    self._create_manager(ou_uuid, manager_uuid, manager_level)
+                else:
+                    logger.error(
+                        f"Manager with uuid: {manager_uuid} has no active engagements and will not be made manager")
+
+            elif ou_manager['uuid'] != manager_uuid:
+                logger.info("Manager for {} should be {}".format(
+                    ou['name'], manager['Navn']))
+
+                self._terminate_manager(ou_manager['relation_uuid'])
+
+                manager_level = self._get_org_level(ou)
+                if has_engagements:
+                    self._create_manager(ou_uuid, manager_uuid, manager_level)
+                else:
+                    logger.error(
+                        f"Manager with uuid: {manager_uuid} has no active engagements and will not be made manager")
+
+            if ou_manager != {} and ou_manager['uuid'] == manager_uuid:
+                # current manager is this ou's manager,
+                # but the manager no longer has active engagements
+                # then remove manager
+                if not has_engagements:
+                    logger.info("Manager for {} is {} and has no engagements. Will be terminated.".format(
+                        ou['name'], manager['Navn']))
+                    self._terminate_manager(ou_manager['relation_uuid'])
+        else:
+            if ou_manager != {}:
+                logger.info("Manager for {} is {} and but no leader association exists. Will be terminated.".format(
+                    ou['name'], ou_manager['Navn']))
+                self._terminate_manager(ou_manager['relation_uuid'])
+
+    def add_employee(self, employee_info):
+        # TODO: add check for data before posting
+
+        engagement_types = {
+            "ML": "55e2f6c9-2dcb-cdc1-556c-d78d7f9e173d",
+            "TL": "b523b991-ece7-0cce-dbcd-7f227e57ad81"
+        }
+
+        org_uuid = self.mh.read_organisation()
+
+        employee_info['org']['uuid'] = org_uuid
+
+        response = self.mh._mo_post('e/create', employee_info, False)
+        if response.status_code == 400:
+            error = response.json()['description']
+            raise requests.HTTPError("Inserting mora data failed")
+        elif response.status_code not in (200, 201):
+            logger.error(
+                'MO post. Response: {}, data'.format(response.text, response)
+            )
+            raise requests.HTTPError("Inserting mora data failed")
+        else:
+            uuid = response.json()
+        return uuid
```

Comment:
Fordi de to andre grene ender i Exceptions ville jeg her droppe `else`.
```suggestion
        uuid = response.json()
        return uuid
```